components/layout/incremental.rs

```diff
@@ -7,6 +7,12 @@ use style::computed_values::float;
 use style::selector_parser::RestyleDamage;
 use style::servo::restyle_damage::{REFLOW, RECONSTRUCT_FLOW};
 
+#[derive(Clone, Copy, PartialEq)]
+pub enum RelayoutMode {
```

Comment:
I believe a comment here saying why is this used and where would be helpful, but apart from this, it looks good.